app/views/course_templates/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing course_template</h1>
+
+<%= render 'form' %>
+
+<%= link_to 'Show', @course_template %> |
```

Comment:
Could we drop the show page and only have the edit page if this is for admins only?